docs/architecture/microservices/implement-resilient-applications/use-httpclientfactory-to-implement-resilient-http-requests.md

```diff
@@ -174,14 +177,17 @@ Up to this point, the code shown is just performing regular Http requests, but t
 
 ## Additional resources
 
-- **Using HttpClientFactory in .NET Core** \
```

Comment:
loc confirmed they fixed the issue on the MT engine so we can use those again - just FYI

Reply:
Oh, great 😊

The trailing two blank spaces are very easy to miss!

Had been removing them 🙁, so back to adding them!